Riot/Modules/Room/AlternateCreation/RoleFilteredSearchController.swift

```diff
@@ -0,0 +1,31 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+class RoleFilteredSearchController: SelectableFilteredSearchController<RoleModel> {
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 0
```

Comment:
This is interim code to ensure that the role view returns an empty set for now, to avoid confusion for testers.  This will be implemented correctly when we can connect to our own back-end and receive role information.